start-as-experiment.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Start as an Experiment
```

Comment:
I think that this pattern may be rediscovering a lot of **Trial Run** from the [_Fearless Change Pattens_](http://www.fearlesschangepatterns.com/).  It's awesome because it means that the content and idea here is solid, but I think it would be good (in the long run) to think about how patterns here relate to what is already documented and printed there.

Reply:
The connection is obvious, indeed. Are you suggesting to add a section _Related Work_ or something similar?

Reply:
I think that would be good for now.  In the long run I'm wondering if we explicitly just start referring people to go read patterns from that book and just focus on any innersource-specific adaptations/examples in our documentation.

Reply:
Added section on related work.